config/task2.json

```diff
@@ -57,71 +58,26 @@
+                    "classes": 2,
                     "coordinates": 4,
-                    "anchors": [
-                        2.896484375,
-                        5.90234375,
-                        4.6875,
-                        9.09375,
-                        7.75390625,
-                        14.59375,
-                        12.8359375,
-                        22.84375,
-                        28.59375,
-                        48.25,
-                        53.875,
-                        109.6875,
-                        109.5625,
-                        155.375,
-                        142.25,
-                        243.75,
-                        209.375,
-                        383.0
-                    ],
-                    "anchor_masks": {
-                        "side52": [
-                            0,
-                            1,
-                            2
-                        ],
-                        "side26": [
-                            3,
-                            4,
-                            5
-                        ],
-                        "side13": [
-                            6,
-                            7,
-                            8
-                        ]
-                    },
+                    "anchors": [],
+                    "anchor_masks": {},
                     "iou_threshold": 0.5,
                     "confidence_threshold": 0.5
                 }
             },
-            "mappings": {
-                "labels": [
-    "person",         "bicycle",    "car",           "motorbike",     "aeroplane",   "bus",           "train",
-    "truck",          "boat",       "traffic light", "fire hydrant",  "stop sign",   "parking meter", "bench",
-    "bird",           "cat",        "dog",           "horse",         "sheep",       "cow",           "elephant",
-    "bear",           "zebra",      "giraffe",       "backpack",      "umbrella",    "handbag",       "tie",
-    "suitcase",       "frisbee",    "skis",          "snowboard",     "sports ball", "kite",          "baseball bat",
-    "baseball glove", "skateboard", "surfboard",     "tennis racket", "bottle",      "wine glass",    "cup",
-    "fork",           "knife",      "spoon",         "bowl",          "banana",      "apple",         "sandwich",
-    "orange",         "broccoli",   "carrot",        "hot dog",       "pizza",       "donut",         "cake",
-    "chair",          "sofa",       "pottedplant",   "bed",           "diningtable", "toilet",        "tvmonitor",
-    "laptop",         "mouse",      "remote",        "keyboard",      "cell phone",  "microwave",     "oven",
-    "toaster",        "sink",       "refrigerator",  "book",          "clock",       "vase",          "scissors",
-    "teddy bear",     "hair drier", "toothbrush"
-               ]
+        "mappings": {
+            "labels": [
+            "strawberries",
+            "strawberry"
```

Comment:
It is question for next detector version, but do we need two classes for strawberry?